tests/FeeSharingCollectorMultiToken.test.js

```diff
@@ -0,0 +1,3051 @@
+            );
+
+            tx = await feeSharingCollector.withdrawFeesFromLBDex();
+
+            let latestLBPair1BalanceSUSD = await SUSD.balanceOf(lbPair1.address);
+            let latestLBPair2BalanceSUSD = await SUSD.balanceOf(lbPair2.address);
+            let latestFeeSharingCollectorBalanceSUSD = await SUSD.balanceOf(
+                feeSharingCollector.address
+            );
+
+            let latestLBPair1BalanceWrbtc = await WrappedNativeToken.balanceOf(lbPair1.address);
+            let latestFeeSharingCollectorBalanceWrbtc = await WrappedNativeToken.balanceOf(
+                feeSharingCollector.address
+            );
+
+            let latestLBPair2BalanceSov = await SOVToken.balanceOf(lbPair2.address);
+            let latestFeeSharingCollectorBalanceSov = await SOVToken.balanceOf(
+                feeSharingCollector.address
+            );
+
+            expect(previousLBPair1BalanceSUSD.toString()).to.equal(
+                latestLBPair1BalanceSUSD.add(lbPair1Amount).toString()
+            );
+            expect(previousLBPair2BalanceSUSD.toString()).to.equal(
+                latestLBPair2BalanceSUSD.add(lbPair2Amount).toString()
+            );
+            expect(previousFeeSharingCollectorBalanceSUSD.toString()).to.equal("0");
+            expect(latestFeeSharingCollectorBalanceSUSD.toString()).to.equal(
+                totalFeeAmount.toString()
+            );
+
+            expect(previousLBPair1BalanceWrbtc.toString()).to.equal(
+                latestLBPair1BalanceWrbtc.add(lbPair1Amount).toString()
+            );
+            expect(previousFeeSharingCollectorBalanceWrbtc.toString()).to.equal("0");
+            expect(latestFeeSharingCollectorBalanceWrbtc.toString()).to.equal(
+                lbPair1Amount.toString()
+            );
+
+            expect(previousLBPair2BalanceSov.toString()).to.equal(
+                latestLBPair2BalanceSov.add(lbPair2Amount).toString()
+            );
+            expect(previousFeeSharingCollectorBalanceSov.toString()).to.equal("0");
+            expect(latestFeeSharingCollectorBalanceSov.toString()).to.equal(
+                lbPair2Amount.toString()
+            );
+        });
+    });
+
+    describe("transferTokens", () => {
+        it("Shouldn't be able to use zero token address", async () => {
+            await protocolDeploymentFixture();
+            await expectRevert(
+                feeSharingCollector.transferTokens(ZERO_ADDRESS, 1000),
+                "FeeSharingCollectorMultiToken::transferTokens: invalid address"
+            );
+        });
+
+        it("Shouldn't be able to transfer zero amount", async () => {
+            await protocolDeploymentFixture();
+            await expectRevert(
+                feeSharingCollector.transferTokens(SOVToken.address, 0),
+                "FeeSharingCollectorMultiToken::transferTokens: invalid amount"
+            );
+        });
+
+        it("Shouldn't be able to withdraw zero amount", async () => {
```

Comment:
Wrong testdescription. Instead of "Shouldn't be able to withdraw zero amount", should say: "Shouldn't be able to withdraw above available"